Source/Android/app/src/main/res/values/strings.xml

```diff
@@ -137,6 +137,8 @@
     <string name="audio_stretch_description">Stretches audio to reduce stuttering. Increases latency.</string>
     <string name="enable_save_states">Enable Savestates</string>
     <string name="enable_save_states_description">WARNING: Savestates may not be compatible with future versions of Dolphin and can make it impossible to create normal saves in some cases. Never use savestates as the only way of saving your progress.</string>
+    <string name="lock_emulation_landscape">Lock screen to landscape</string>
+    <string name="lock_emulation_landscape_desc">Some touch controls will require additional tweaking if played portrait</string>
```

Comment:
This might just be me, but *if played portrait* reads somewhat odd here. Was this intended to be something like:

> Some touch controls will require additional tweaking if played in portrait mode

?